Sources/PointFree/Session.swift

```diff
@@ -50,11 +50,122 @@ extension URLRequest {
   }
 }
 
-public struct Session: Codable, Equatable {
+public struct Session: Equatable {
   public var flash: Flash?
-  public var userId: User.Id?
+  public var user: User?
 
-  public static let empty = Session(flash: nil, userId: nil)
+  public static let empty = Session(flash: nil, user: nil)
+
+  public var userId: Models.User.Id? {
+    get {
```

Comment:
Minor: nesting you could kill...

Reply:
ah yeah, I had a deprecated `set` for backwards compatibility but was able to kill it last minute